beers/test/factories.py

```diff
@@ -0,0 +1,19 @@
+import factory
+
+from beers.models import BeerStyle
+
+class BeerStyleFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = BeerStyle
+
+    name = 'Super Light Beer'
+    revision = '2015'
+    bjcp_class = 'Beer'
+    category = '42'
```

Comment:
use `factory.Sequence` for the category and subcategory so we get variations here.